src/translations/es.ts

```diff
@@ -0,0 +1,182 @@
+    "point-outlier": "{x}, {y}, {index} de {count}",
+    "point-xhl": "{x}, {high} - {low}",
+    "point-xhl-outlier": `{x}, {high} - {low}, con {count, plural, 
+        =0 {Sin valores atípicos} 
+        one {{count} valor atípico} 
+        other {{count} valores atípicos}
+    }`,
+
+    "info-open": "Abrir info",
+    "info-title": "Info",
+    "info-notes": "Notas",
+
+    "kbmg-title": "Administrador de teclado",
+    "key-point-next": "Ir al punto siguiente",
+    "key-point-prev": "Ir al punto anterior",
+    "key-play-fwd": "Reproducir a la derecha",
+    "key-play-back": "Reproducir a la izquierda",
+    "key-play-cancel": "Cancelar reproducción",
+    "key-group-prev": "Ir al grupo anterior",
+    "key-stat-prev": "Navegar a estadística anterior",
+    "key-group-next": "Ir al grupo siguiente",
+    "key-stat-next": "Navegar a la siguiente estadística",
+    "key-group-first": "Ir al primer grupo",
+    "key-group-last": "Ir al último grupo",
+    "key-hier-root": "Ir a la raíz",
+    "key-play-fwd-group": "Reproducir hacia adelante por los grupos",
+    "key-play-back-group": "Reproducir hacia atrás por los grupos",
+    "key-point-first": "Ir al primer punto",
+    "key-point-last": "Ir al último punto",
+    "key-replay": "Reproducir de nuevo",
+    "key-select": "Seleccionar ítem",
+    "key-tenth-prev": "Retroceder una décima",
+    "key-tenth-next": "Adelantar una décima",
+    "key-level-min": "Ir al menor valor del nivel",
+    "key-level-max": "Ir al mayor valor del nivel",
+    "key-group-min": "Ir al menor valor del grupo",
+    "key-group-max": "Ir al mayor valor del grupo",
+    "key-chart-min": "Ir al menor valor del gráfico",
+    "key-chart-max": "Ir al mayor valor del gráfico",
+    "key-level-decr": "Bajar un nivel",
+    "key-level-incr": "Subir un nivel",
+    "key-speed-incr": "Acelerar",
+    "key-speed-decr": "Ralentizar",
+    "key-monitor-toggle": "Cambiar modo de pantalla",
+    "key-dialog-help": "Abrir diálogo de ayuda",
+    "key-dialog-options": "Abrir diálogo de opciones",
+
+    "options-title": "Opciones",
+    "options-frontmatter":
+        "Al navegar este gráfico algunos sonidos pueden reusltar muy altos o muy bajos para que los escuche. Adicionalmente, si desea expandir el rango de sonidos disponibles, deslice los controles para ajustar el rango de sonido:",
+    "options-hertz-lower": "Hertz más bajo",
+    "options-hertz-upper": "Hertz más alto",
+    "options-speed-label": "Play speed (aka, press 'Q' and 'E')",
+    "options-set-global":
+        "Guardar mis opciones para otros gráficos en esta página",
+    "options-use-continuous": "Usar modo continuo",
+    "options-continuous-descr":
+        "El modo continuo cambia cómo se reproducen los valores al presionar Shift+Home y Shift+End",
+    "options-point-labels": "Mostrar etiquetas del punto",
+    "options-point-labels-before": `valores anteriores (eg: "California, 163,696 square miles, 39 million people" )`,
```

Comment:
Do you mind translating the "square miles" and "million people", as well? Feel free to convert to km2, if you think that makes more sense.